src/modules/sessions/sessions.route.ts

```diff
@@ -29,6 +29,17 @@ const controller = new SessionsController();
  * @property {string} updatedAt - updated at
  */
 
+/**
+ * @typedef {object} CreateSession
+ * @property {string} name.required - Name of the session
+ * @property {string} location.required - location
+ * @property {string} sessionTime.required - Time of session
```

Comment:
please add sessionDate you could base on this

https://github.com/Zeff01/formapp-be/blob/ccf2d30a8f7ffc6937558050b4db50d24ce5b6bd/src/modules/sessions/sessions.service.ts#L102-L106